native/chat/chat-thread-list.react.js

```diff
@@ -76,18 +80,22 @@ class ChatThreadList extends React.PureComponent<Props, State> {
   flatList: ?FlatList<Item>;
 
   componentDidMount() {
-    const tabNavigation: ?TabNavigationProp<
+    let tabNavigation: ?TabNavigationProp<
```

Comment:
The type here is wrong. The typechecker isn't complaining because it doesn't know how to check `dangerouslyGetParent`, but we still shouldn't be typing things incorrectly. The grandparent is a `TabNavigationProp` but the parent is not. Can you use two different variables here, and type each of them correctly?